app/controllers/books/filter_controller.rb

```diff
@@ -1,8 +1,13 @@
 module Books
   class FilterController < ApplicationController
+    PER_PAGE = 20
```

Comment:
Maybe you can try to add a Kaminari default configs here: https://github.com/amatsuda/kaminari#general-configuration-options
and define the default number of "default_per_page" to 20. Doing it you will also be able to remove those ".per(PER_PAGE)" piece of code from both controllers.